src/components/layout/nav/Navbar.scss

```diff
@@ -1,5 +1,9 @@
 @import "../../common/variables";
 
+$navbarIndex: 102;
```

Comment:
it's too late to fix this but I'll say in general if you're using z indexes something weird is going on, especially like this. that it to say I almost never have to use z indexes except in a few rare cases. be more careful of this in the future

Reply:
you should add some fairly descriptive comments above this explaining why this is needed. someone who knows nothing about your code should be able to understand by reading said comments

Reply:
Hmm, anladım.